### PR TITLE
Homework9: Mariya Simonenko

### PR DESCRIPTION
**Pull request by Mary-si**

### CODE REVIEW

File: homework9_candles.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+# Проверка
+print(solution(5, 2))
+print(solution(1, 2))
+print(solution(15, 5))
+print(solution(12, 2))
+print(solution(6, 4))
+print(solution(13, 5))
+print(solution(2, 3))
```
Comment (Stml89) [inline homework9_candles.py:52]: remove line#46-52

File: homework9_symbo1.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            if result:
+                result.pop()
+        else:
+            result.append(s[i])
+        i += 1
+    return "".join(result)
+
+
+# Проверка
+print(symbols("a#bc#d"))
```
Comment (Stml89) [inline homework9_symbo1.py:32]: use `assert` instead of `print`

File: homework9_symbol.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# "abcde" == "abcde"
+# "aaabbdefffff" == "a3b2def5"
+
+
+def count_s(a):
+    """Преобразование из "cccbba" в c3b2a"""
+    symbol = a[0]
+    count = 0
+    sum_sym = ""
+    for [_, s] in enumerate(a):
```
Comment (Stml89) [inline homework9_symbol.py:19]: since you are not using `index`, you can iterate over a string:
`
for i in a:
...
`

File: homework9_symbol.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        if symbol == s:
+            count += 1
+        if symbol != s:
```
Comment (Stml89) [inline homework9_symbol.py:22]: `elif` here, symbol can be the the on not?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework9_symbo1.py (hunk 2 above)
Comment (Mary-si) [reply homework9_symbo1.py:32]: поправила

File: homework9_symbol.py (hunk 4 above)
Comment (Mary-si) [reply homework9_symbol.py:22]: поправила

**Review by Stml89 — APPROVED** [review]

**Pull request merged**